docker-entrypoint.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+# Exit immediately if a command exits with a non-zero status.
+set -e
+
+# Symlink /tmp/node_modules if necessary
+if [ ! -h 'node_modules' ]; then
```

Comment:
Looks like it needs a `-d` instead of `-h` for the shell inside of the container. 

`docker-compose up` with`-h` fails for me the second time with:

```shell
$ docker-compose up
Starting irisfrontend_web_1
Attaching to irisfrontend_web_1
web_1  | Symlink preinstalled node_modules from /tmp
web_1  | ln: failed to create symbolic link '/app/node_modules/node_modules': File exists
irisfrontend_web_1 exited with code 1
```